packages/react-link/src/Link.stories.tsx

```diff
@@ -1,13 +1,19 @@
 import * as React from 'react';
-import { Stack, IStackTokens, Text } from '@fluentui/react';
-import { Link, LinkProps } from '@fluentui/react-link';
+import { Link } from './components/Link/Link';
+import { LinkProps } from './components/Link/Link.types';
 
-const linksStackTokens: IStackTokens = {
-  childrenGap: 10,
+const flexStyles: React.CSSProperties = {
```

Comment:
**suggestion:** can we use makeStyles going forward for consistency 🙏 ?

Reply:
For sure, I will change it. Thanks.